package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "slugg",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Please remove this, I use `npm version major|minor|patch` to do releases which keeps tags and package.json version in sync.